### PR TITLE
Rename Event resource to SeamEvent

### PR DESCRIPTION
**Pull request by andrii-balitskyi**
Closes https://github.com/seamapi/ruby-next/issues/78

### CODE REVIEW

File: lib/seam/routes/clients/events.rb [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 module Seam
   module Clients
-    class Events < BaseClient
+    class SeamEvents < BaseClient
```
Comment (razor-x) [inline lib/seam/routes/clients/events.rb:5]: This can stay `Events`. It's only the `Event` class we need to worry about.

Comment (andrii-balitskyi) [reply lib/seam/routes/clients/events.rb:5]: Gotcha. Should we change it back on [Python SDK](https://github.com/seamapi/python/blob/370520b1ab8ea88a4375f8bf69215df78d004f3c/seam/routes/events.py#L6) as well?

Comment (razor-x) [reply lib/seam/routes/clients/events.rb:5]: Yes, sure. The class isn't really part of the public API of the SDK so it should be ok to rename.

Comment (andrii-balitskyi) [reply lib/seam/routes/clients/events.rb]: c974a700b5004432bce61c824f384a4a3b993108

File: lib/seam/helpers/action_attempt.rb
Comment (andrii-balitskyi) [inline lib/seam/helpers/action_attempt.rb:1]: These are the changes from this [PR](https://github.com/seamapi/ruby-next/pull/102)

**Review by razor-x — APPROVED** [review]

**Pull request merged**